include/lsst/afw/table/Schema.h

```diff
@@ -417,7 +417,7 @@ class SubSchema {
     /**
      *  @brief Implicit conversion to the appropriate Key type.
      *
-     *  Implicit conversion operators cannot be translated to Python.  Instead, the SWIG
+     *  Implicit conversion operators cannot be translated to Python.  Instead, the Python
      *  wrappers provide an equivalent asKey() method.
      */
     template <typename T>
```

Comment:
Since implicit conversions *can* be replicated with pybind11, will this be removed in the future?